database/buildSchema/29_review_response.sql

```diff
@@ -14,25 +16,32 @@ CREATE TABLE public.review_response (
 	original_review_response_id integer references public.review_response(id),
 	review_id integer references public.review(id),
     time_created timestamptz default current_timestamp,
+	is_visible_to_applicant boolean default false,
+	template_element_id integer references public.template_element,
+	recommended_applicant_visibility public.review_response_recommended_applicant_visibility default 'Original Response Not Visible to Applicant',
 	status public.review_response_status default 'Draft'
```

Comment:
If the field was named 'is_recommended_visible_to_applicant` then `true`/`false` would be okay, wouldn't it? (sorry, booleans are just way easier to work with in the front-end, but no worries if you'd prefer this)